gsd-schema/osv-gsd/validation/schema-osv-gsd-1.0.1.json

```diff
@@ -311,7 +311,9 @@
   },
   "required": [
     "id",
-    "modified"
+    "modified",
+    "schema_type",
```

Comment:
If we're adding required fields, it might make sense to also require things like `schema_version`